### PR TITLE
Support onnx export with previous OpSet version

### PR DESCRIPTION
**Pull request by wangyems**
Feature request https://github.com/dotnet/machinelearning/issues/5171

### CODE REVIEW

**Comment by wangyems** [comment]
Haven't figure out a god place to put the comments(Estimator requires OpSet 11) for featureselectionestimator

File: src/Microsoft.ML.Transforms/Text/TextNormalizing.cs [hunk 1]
```diff
@@ -215,6 +215,7 @@ public Mapper(TextNormalizingTransformer parent, DataViewSchema inputSchema)
 
             public bool CanSaveOnnx(OnnxContext ctx) => (_parent._keepDiacritics && _parent._keepNumbers && _parent._keepPunctuations);
 
+            //This Estimator requires OpSet 11
             public void SaveAsOnnx(OnnxContext ctx)
```
Comment (harishsk) [inline src/Microsoft.ML.Transforms/Text/TextNormalizing.cs:219]: The TextNormalizingEstimator has in its doc-comments a field for whether it is exportable to Onnx. There you can specify the required version number.

File: test/Microsoft.ML.Tests/OnnxConversionTest.cs [hunk 2]
```diff
@@ -172,7 +172,7 @@ public void KmeansOnnxConversionTest()
             var model = pipeline.Fit(data);
             var transformedData = model.Transform(data);
 
-            var onnxModel = mlContext.Model.ConvertToOnnxProtobuf(model, data);
+            var onnxModel = mlContext.Model.ConvertToOnnxProtobuf(model, data, 11);
```
Comment (Lynx1820) [inline test/Microsoft.ML.Tests/OnnxConversionTest.cs:175]: Will the opset for these tests always be 11? If it's likely to change, would it make sense to make a variable with the current opset?

Comment (wangyems) [reply test/Microsoft.ML.Tests/OnnxConversionTest.cs:175]: We will change this implementation, just ignore it. The initial thought was to set the default opset version to 9 and explicitly set new version of opset if the transformer requires newer opset version

**Pull request closed without merging**